.github/workflows/welcome-bot.yml

```
@@ -0,0 +1,30 @@
+name: WelcomeBot
+
+on:
+  pull_request_target:
+    branches: [main]
+    types: [opened]
+
+permissions:
+  pull-requests: write
+
+jobs:
+  welcome:
+    name: Welcome First Time Contributors
```

Nit that I got wrong in the docs version 🫣
```suggestion
    name: Welcome First-Time Contributors
```